src/views/received_sms_list/SmsTable.js

```diff
@@ -43,7 +42,11 @@ const SmsTable = ({ messages, pager, selectedIds, setSelectedIds }) => {
             <TableHead>
                 <TableRowHead>
                     <TableCellHead>
-                        <Checkbox checked={allSelected} onChange={toggleAll} />
+                        <Checkbox
+                            disabled={messages.length === 0}
+                            onChange={toggleAll}
+                            checked={allSelected}
+                        />
```

Comment:
This change is unrelated right? Not a huge deal if it is, but as a reviewer I'd prefer that in a separate commit for ease of reviewing (and finding the change later, if necessary).

Reply:
That's right it was just part of trying to make components between both sections more similar. I should have moved these kind of changes to a different commit/PR for sure :sweat_smile: